src/Track.h

```diff
@@ -1486,16 +1499,15 @@ class AUDACITY_DLL_API TrackList final
    template<typename TrackKind>
       TrackKind *Add( const std::shared_ptr< TrackKind > &t )
          { return static_cast< TrackKind* >( DoAdd( t ) ); }
-
-   /** \brief Define a group of channels starting at the given track
-   *
-   * @param track and (groupSize - 1) following tracks must be in this
-   * list.  They will be disassociated from any groups they already belong to.
-   * @param groupSize must be at least 1.
-   * @param resetChannels if true, disassociated channels will be marked Mono.
+   
+   //! Removes link to the next track
+   void UnlinkChannels(Track& first);
+   /** \brief Links track with the next one in the list.
+   * @param first and the following must be in this list.
+   * @param aligned if true, the link type will be set to Track::LinkType::Aligned,
+   * and Track::LinkType::Group otherwise
    */
-   void GroupChannels(
-      Track &track, size_t groupSize, bool resetChannels = true );
+   void LinkChannels(Track& first, bool aligned);
```

Comment:
Really, no.  I don't like this part.

Leave as it was:  one function, not two functions, with an interface equipped to make channel groups of any size, even though this isn't yet fully implemented.  Just add an argument for "alignment" or not.

Update function comments to say "alignment" has no effect when there is just one channel; and that grouping past two is not necessarily implemented yet, but the interface is presented.